docs/topics/modules/api-get-component-analyses-response.adoc

```diff
@@ -0,0 +1,46 @@
+[id="api_get_component_analyses_response"]
+= GET /component-analyses/{ecosystem}/{component name}/{component version}
+
+== Purpose
+Fetches various data points for a selected component.
+
+IMPORTANT: You require an {osio} authentication token to invoke this method.
```

Comment:
This doesn't seem like an 'important' admonition...

* note -- a reminder (such as this case)
* important -- potential breakage if not heeded
* warning -- danger of loss of data